### PR TITLE
A doc fix for azure_rm_virtualmachinescaleset

### PR DESCRIPTION
**Pull request by mmyyrroonn**
##### SUMMARY

Ref:
https://github.com/Azure/azure_preview_modules/issues/259

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_virtualmachinescaleset

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by yungezz** [comment]
@zikalino pls help to review

**Comment by Fred-sun** [comment]
@zikalino Please give a review when you're available! Thanks a lot!

**Comment by Fred-sun** [comment]
@zikalino

**Review by yungezz — APPROVED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py [hunk 1]
```diff
@@ -277,6 +280,7 @@
     vm_size: Standard_DS1_v2
     capacity: 2
     virtual_network_name: testvnet
+    upgrade_policy: Manual
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py:283]: how the test passed before?

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @MyronFanQiu for the Ansible documentation fix!

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py (hunk 1 above)
Comment (mmyyrroonn) [reply lib/ansible/modules/cloud/azure/azure_rm_virtualmachinescaleset.py:283]: I checked the integration test. All tests are provided this parameter `upgrade_policy`.

**Pull request merged**